R/pkgdown.R

```diff
@@ -30,25 +29,31 @@ use_pkgdown <- function(config_file = "_pkgdown.yml", destdir = "docs") {
   check_is_package("use_pkgdown()")
   check_installed("pkgdown")
 
-  use_build_ignore(c(config_file, destdir))
-  use_build_ignore("pkgdown")
+  use_build_ignore(c(config_file, destdir, "pkgdown"))
   use_git_ignore(destdir)
 
-  ui_todo("
-    Record your site's {ui_field('url')} in the pkgdown config file \\
-    (optional, but recommended)")
```

Comment:
Do you really think we should omit all nudges about this? As it stands, we just silently put this into the config and open the file:

```
url: ~
```

That's not OK, right? Are we hoping the user also realizes they need to fill this in?

Reply:
`~ is fine for a default because it'll be read as `NULL`. 

I thought we also nudged elsewhere?

Reply:
In `use_pkgdown_github_pages()`, when dealing with our own stuff (repo belongs to tidyverse or r-lib orgs), we actually insert the (future?) URL into pkgdown config, because we know what it will be. I think that is what you are thinking of.

But for the general population, I think this is our only nudge, so suspect it should stay.